pd-configuration-file.md

```diff
@@ -92,6 +92,14 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 强制让该 PD 以一个新集群启动，且修改 raft 成员数为 1。
 + 默认值：false
 
+### `tso-update-physical-interval` <span class="version-mark">从 v6.4.0 版本开始引入</span>
```

Comment:
```suggestion
### `tso-update-physical-interval`
```
`tso-update-physical-interval` is introduced in https://github.com/tikv/pd/pull/3027.

Reply:
Fixed, PTAL~